### PR TITLE
Added fake incident reports

### PR DESCRIPTION
**Pull request by sandlerben**

### CODE REVIEW

**Comment by sandlerben** [comment]
r @nancywong

File: app/models/incident_report.py [hunk 1]
```diff
@@ -25,3 +26,34 @@ class IncidentReport(db.Model):
...
+
+        agencies = Agency.query.all()
+
+        seed()
+        for i in range(count):
+            l = Location(original_user_text=forgery_py.address.street_address())
+            u = IncidentReport(
+                vehicle_id=forgery_py.basic.text(length=6, spaces=False),
+                license_plate=forgery_py.basic.text(at_least=6, spaces=False)
+                if choice([True, False]) else '',
```
Comment (rileynwong) [inline app/models/incident_report.py]: can you explain what this line does? (`if choice`...) or maybe add a short comment?

File: app/models/incident_report.py [hunk 2]
```diff
@@ -25,3 +26,34 @@ class IncidentReport(db.Model):
...
+        from random import seed, choice, randint
+        from datetime import timedelta
+        import forgery_py
+
+        agencies = Agency.query.all()
+
+        seed()
+        for i in range(count):
+            l = Location(original_user_text=forgery_py.address.street_address())
+            u = IncidentReport(
```
Comment (rileynwong) [inline app/models/incident_report.py]: very small/can ignore, but I would use r instead of u as the temp variable (short for 'report')

**Comment by rileynwong** [comment]
otherwise lgtm!

**Comment by sandlerben** [comment]
Thanks @nancywong. Let's merge #6 before we merge #10 or #12

**Pull request merged**